contracts/test/unit/CustomVerifier.test.ts

```
@@ -0,0 +1,411 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { TestCustomVerifier, CustomVerifier } from "../../typechain-types";
+
+export const AttestationId = {
+  E_PASSPORT: "0x0000000000000000000000000000000000000000000000000000000000000001",
+  EU_ID_CARD: "0x0000000000000000000000000000000000000000000000000000000000000002",
+} as const;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)